### ISSUE TITLE
Gradient-augmented Supervised Learning of Optimal Feedback Laws Using State-dependent Riccati Equations. (arXiv:2103.04091v1 [math.OC])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#4456]
Gradient-augmented Supervised Learning of Optimal Feedback Laws Using State-dependent Riccati Equations. (arXiv:2103.04091v1 [math.OC])<br>
https://ift.tt/30oYEV5<br>
<p>A supervised learning approach for the solution of large-scale nonlinear stabilization problems is presented. A stabilizing feedback law is trained from a dataset generated from State-dependent Riccati Equation solves. The training phase is enriched by the use gradient information in the loss function, which is weighted through the use of hyperparameters. High-dimensional nonlinear stabilization tests demonstrate that real-time sequential large-scale Algebraic Riccati Equation solves can be substituted by a suitably trained feedforward neural network.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
